yark-pages/src/lib/store.ts

```diff
@@ -21,6 +24,9 @@ yarkStore.subscribe((value) => {
  * @returns Relevant initial Yark store
  */
 function yarkStoreInitial(): YarkStore {
+    // workaround for sveltekit ssr for some reason running during development
```

Comment:
Capitalize this and link to this pr (`see #111 <https://github.com/Owez/yark/pull/111>`)